charts/yandex-csi-controller/values.yaml

```diff
@@ -0,0 +1,172 @@
+
+ycsi:
+  cloud_secret:
```

Comment:
Let's just provide this as a string blob that contains the whole ServiceAccount JSON. No reason to split it whatsoever.

Reply:
fix it